prijateli_tree/app/main.py

```diff
@@ -20,8 +19,6 @@
 )
 
 
-Base.metadata.create_all(bind=engine)
```

Comment:
Removed this line so that we rely on alembic to create the database tables, and not FastAPI.